internal/runtime/config_test.go

```diff
@@ -54,6 +54,8 @@ var _ = Describe("Viper to Runtime Config", func() {
 		expectedRuntimeCfg.Channel = "mychannel"
 		baseViperCfg.Set("indeximage", "myindeximage")
 		expectedRuntimeCfg.IndexImage = "myindeximage"
+		baseViperCfg.Set("csv_timeout", 180)
+		expectedRuntimeCfg.CSVTimeout = 180
```

Comment:
These should all reference the aforementioned constant.

Reply:
Added new defaults file for this.